conans/test/functional/command/config_install_test.py

```diff
@@ -521,3 +522,53 @@ def get_zip():
         self.client.run("config install http://localhost:%s/myconfig.zip" % http_server.port)
         self.assertIn("Unzipping", self.client.out)
         http_server.stop()
+
+    def test_config_install_sched_error(self):
+        client = TestClient()
+        client.run('config set general.config_install_interval=1s')
+        # Any conan invocation will fire the configuration error
+        client.run('install .', assert_error=True)
+        self.assertIn("ERROR: Incorrect definition of config_install_interval: 1s", client.out)
+
+    def test_config_install_sched_file(self):
+        """ Conan must execute config once when config_install_interval is configured
+        """
+        folder = temp_folder(path_with_spaces=False)
+        conanconf = textwrap.dedent("""
+            [general]
+            config_install_interval = 5m
+            """)
+        save_files(folder, {"conan.conf": conanconf})
+        client = TestClient()
+
+        # 1 - config install can be executed without restriction
+        client.run('config install "%s"' % folder)
+        self.assertIn("Processing conan.conf", client.out)
+        content = load(client.cache.conan_conf_path)
+        self.assertEqual(1, content.count("config_install_interval"))
```

Comment:
This would be true if the file contains

```ini
# config_install_interval = 1h
```

which is the default.

Reply:
This is counting the name of occurrences of the literal "config_install_interval", not the value

Reply:
It should. The default content of `conan.conf` contains `1` appearance of the `config_install_interval` too. I suppose that we want to check that the command runs successfully and it contains the content of the configuration in `folder`, not the default one.

Reply:
Yes, please @uilianries add that check

Reply:
> # config_install_interval = 1h

No, when is commented the content.count returns 0.

Reply:
It shouldn't matter if commented or not, it is a literal string search ``content = load(client.cache.conan_conf_path)``

The goal is to validate that the new conan.conf contains `` config_install_interval = 5m``, because it was installed from the folder